lib/src/sql/param.rs

```diff
@@ -67,6 +67,9 @@ impl Param {
 				Some(v) => v.compute(ctx, opt, txn, doc).await,
 				// The param has not been set locally
 				None => {
+					// Check that namespace and database are set to prevent a panic
+					opt.valid_for_ns()?;
```

Comment:
Just realised, we shouldn't need this line, as the `opt.valid_for_db()?;` line calls this internally.
```suggestion
```